lib/index.coffee

```diff
@@ -68,33 +68,21 @@ compare = (a, b) ->
 
 # arr is array of arrays to be merged
 # modifies arrs, TODO: offsets
-exports.merge = merge = (arrs, comp) ->
-	comp ?= (a, b) -> a - b
-	ret = []
-
-	totalLength = _.sum(([arr.length for arr in arrs])...)
-
-	for i in [0...totalLength]
-		min = null
-		minArrayIndex = null
-		for arr, index in arrs when arr.length isnt 0
-			if min is null or comp(arr[0], min) < 0
-				min = arr[0]
-				minArrayIndex = index
-		ret.push(min)
-		arrs[minArrayIndex].shift()
-
-	return ret
+exports.merge = merge = (arrs, comp = 0) ->
```

Comment:
`comp` needs to be a comparison function. I meant to pass the compare function defined above which compares based on two fields but missed it.